cms/src/main/java/uk/nhs/digital/externalstorage/workflow/action/ExternalFileUnpublishAction.java

```diff
@@ -0,0 +1,33 @@
+package uk.nhs.digital.externalstorage.workflow.action;
+
+import org.apache.commons.scxml2.SCXMLExpressionException;
+import org.apache.commons.scxml2.model.ModelException;
+import org.onehippo.cms7.services.HippoServiceRegistry;
+import org.onehippo.repository.documentworkflow.action.AbstractDocumentTaskAction;
+import uk.nhs.digital.externalstorage.s3.S3Connector;
+import uk.nhs.digital.externalstorage.workflow.task.ExternalFileUnpublishTask;
+
+public class ExternalFileUnpublishAction extends AbstractDocumentTaskAction<ExternalFileUnpublishTask> {
```

Comment:
Same as other action, don't think you need the parameters for explicit variants and don't pass parameters into the constructor.